draft-packages/badge/.npmignore

```diff
@@ -0,0 +1,6 @@
+**/*.ts
```

Comment:
This is per package?! Gross :'(

Reply:
Do you need to ignore `.npmignore` too? Or is that just done by default?

Reply:
I think it needs to be committed, otherwise the pipeline won't be able to read it

Reply:
Yeah definitely committed, but does it need to exist in the dist? This file tells it what to include in the distribution right?

Reply:
Lol actually, this files definition works fine:
```
    "**/*.js",
    "**/*.map.js",
    "**/*.d.ts"
```

Reply:
True! As long as we haven't lost anything in between :)